src/parser/transform/expression/transform_function.cpp

```diff
@@ -365,6 +365,13 @@ unique_ptr<ParsedExpression> Transformer::TransformFuncCall(duckdb_libpgquery::P
 		expr->case_checks.push_back(std::move(check));
 		expr->else_expr = std::move(children[2]);
 		return std::move(expr);
+	} else if (lowercase_name == "*") {
```

Comment:
I think this needs to be called `unpack` or `columns_unpacked` or so.

what happens if we call this function directly now, e.g.:

```sql
SELECT "*"(42)
```